libs/services/designer-client-services/src/lib/standard/search.ts

```diff
@@ -125,3 +128,22 @@ function filterStateful(operation: DiscoveryOperation<BuiltInOperation> | Connec
     ? operation.properties.capabilities.includes('Stateful') || !operation.properties.capabilities.includes('Stateless')
     : operation.properties.capabilities.includes('Stateless') || !operation.properties.capabilities.includes('Stateful');
 }
+
+function filterAPIConnectors(rawConnectors: Connector[], isAzureConnectorsEnabled: boolean): Connector[] {
+  return rawConnectors.filter((rawConnector: Connector) => {
+    return !equals(rawConnector.name, 'dataOperationNew') && (!needsAzureConnection(rawConnector) || isAzureConnectorsEnabled);
```

Comment:
What's the situation that was causing this exception to be needed?  Do we not want to see that connector when Azure operations are disabled?

Reply:
There is a better explanation on this https://github.com/Azure/LogicAppsUX/issues/2836. But basically, for local development cx can decide to create workflows that don't require connections with remote resources. This implies that there is no subscription id involved.

Reply:
I mean specifically for the `!equals(rawConnector.name, 'dataOperationNew')` part of that function, that connector shouldn't require an azure connection, is there a reason we're hiding that connector specifically?